src/client/styles/components/_button.scss

```diff
@@ -15,12 +15,12 @@
   box-shadow: none;
   display: block;
   float: left;
-  line-height: 56px;
+  //line-height: 56px;
```

Comment:
Git will keep a history of changes that can also be easily viewed on GitHub. Removing of commented code can easily be forgotten so better to remove at time of changing if the changes are necessary.